index.d.ts

```diff
@@ -0,0 +1,21 @@
+import Koa = require("koa");
```

Comment:
@brunokrebs ah, just re-read that post, you are right, they are even saying this explicitly:

> Our package exposes declarations from each of those, so any user of our browserify-typescript-extension package needs to have these dependencies as well. For that reason, we used "dependencies" and not "devDependencies", because otherwise our consumers would have needed to manually install those packages. If we had just written a command line application and not expected our package to be used as a library, we might have used devDependencies.